puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-update-jobs.yml

```diff
@@ -0,0 +1,10 @@
+---
+- job:
+    name: 'foreman-update-jobs'
```

Comment:
Thinking something more descriptive here perhaps, in the sense of adding knowledge that it's related to foreman-infra or jenkins-jobs